src/index.html

```diff
@@ -15,8 +15,91 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <!-- <h1>Moyo header</h1> -->
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="Moyo Company Logo"
+          class="img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="is-active nav__link"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#laptops-computers"
```

Comment:
You have used spaces in the `href` attribute of the `<a>` tag. The `href` attribute should not contain any spaces. Spaces can cause the link to break or not function as expected. Instead of spaces, use hyphens (-) or underscores (_) to separate words in the URL.